utils/visualizer/qviz/content_loader.py

```diff
@@ -179,9 +180,7 @@ def load_single_log_file(file_path: str, revision_id: str) -> (list[dict], set[s
         # RemoveFile entry
         elif 'remove' in action_file:
             remove_file = action_file['remove']
-            _id = remove_file['tags']['revision']
-            if _id == revision_id:
-                remove_paths.add(remove_file['path'])
+            remove_paths.add(remove_file['path'])
```

Comment:
It is removed here because RemoveFiles have their corresponding AddFiles from previous logs, there's no way you add an AddFile and then remove it in the same transaction. So, all removes are gathered together so we can remove their AddFiles together at the end.